src/Core/src/Platform/Android/ButtonExtensions.cs

```diff
@@ -10,6 +11,35 @@ namespace Microsoft.Maui
 {
 	public static class ButtonExtensions
 	{
+		public static void UpdateStrokeColor(this MaterialButton nativeButton, IButtonStroke buttonStroke)
+		{
+			var stroke = buttonStroke.StrokeColor?.ToNative();
+
+			if (stroke is not null)
+			{
+				var states = new int[][] {
+					new int[] { Android.Resource.Attribute.StateEnabled },
+					new int[] {-Android.Resource.Attribute.StateEnabled },
+					new int[] {-Android.Resource.Attribute.StateChecked },
+					new int[] { Android.Resource.Attribute.StatePressed }
+				};
+				var c = (int)stroke;
+				nativeButton.StrokeColor = new ColorStateList(states, new int[] { c, c, c, c });
+			}
```

Comment:
Should this be moved to the ColorExtensions? I recall some static `States` property. I think we should move this there too and it can be used all over.